### PR TITLE
fix(clearFix): Fixes an issue with content being blank.

### PR DESCRIPTION
**Pull request by bhough**
Fixes an issue with clearFix where `content` was blank due to the empty quotes being parsed as a null value.

Addresses: #140

### CODE REVIEW

**Comment by bhough** [comment]
@mxstbr @nikgraf confirmed this is an issue with other libraries and this change fixes it with all of them, ready for review.

File: docs/assets/polished.js [hunk 1]
```diff
@@ -1042,7 +1042,7 @@ var functionsMap = {
 /** */
 
 /**
- * String to represent common easing functions as demonstrated here: (github.com/jaukia/easie).
+ * String to represent commong easing functions as demonstrated here: (github.com/jaukia/easie).
```
Comment (mxstbr) [inline docs/assets/polished.js]: Typo

Comment (bhough) [inline docs/assets/polished.js]: Odd that this is here, but not in my actual changes? Maybe this hasn't been built in awhile?

**Comment by bhough** [comment]
@mxstbr fixed.

**Pull request merged**